packages/app/src/navigation/SidebarNavigationRow.vue

```diff
@@ -48,6 +47,14 @@
       >
         {{ name }}
       </span>
+      <span
```

Comment:
What about pulling this out into its own component?  Any reason that it could not stand on its own?

Reply:
I tried to do this but I couldn't get the animation to work properly due to the component getting destroyed. It's not a lot of logic and makes sense in the context of the row so I'm ok with how it is.